build/electrical/index.md

```diff
@@ -154,6 +154,7 @@ Follow the links to the wiring configuration guides specific to your printer and
 * [V2 - SKR 1.3](./v2_skr13_wiring.md)
 * [V2 - SKR 1.4](./v2_skr14_wiring.md)
 * [V2 - FLYboard FLYF407ZG](./v2_flyf407zg_wiring.md)
+* [V2 - Fysetc Spider(./v2_spider_wiring.md)
```

Comment:
It looks like a ']' is missing, it should be
`* [V2 - Fysetc Spider](./v2_spider_wiring.md)`
instead.